public/app/components/codeEditor/codeEditor.js

```diff
@@ -27,13 +27,30 @@ angular.module('kibibitCodeEditor')
     SettingsService) {
 
     var vm = this;
+    var editor;
+    var editorSettings = SettingsService.settings.editorSettings;
+
+    var init = function(settings) {
+      editor.setOptions({
+        'wrap': settings.lineWrap,
+        'mode': 'ace/mode/' + settings.syntaxMode,
+        'theme': 'ace/theme/' + settings.theme,
+        'tabSize': settings.tabWidth,
+        'fontSize': settings.fontSize,
+        'showGutter': settings.isGutter,
+        'useSoftTabs': settings.isSoftTabs,
+        'showPrintMargin': settings.ruler
+      });
+    };
 
     // initialize the editor session
     vm.aceLoaded = function(_editor) {
+      editor = _editor;
```

Comment:
nit: If you  start with `editor = _editor`, I think it's prettier to continue the rest of the function with `editor` instead of `_editor`